modules/nw-ovn-k-adminnetwork-policy.adoc

```diff
@@ -63,7 +63,7 @@ spec:
             custom-anp: tenant-1
 ----
 <1> Specify a name for your ANP.
-<2> The `spec.priority` field supports a maximum of 100 ANP in the values of 0-99 in a cluster. The lower the value the higher the precedence. Creating `AdminNetworkPolicy` with the same priority creates a nondeterministic outcome.
+<2> The `spec.priority` field supports a maximum of 100 ANP in the values of 0-99 in a cluster. The lower the value the higher the precedence. Because there is no guarantee which policy will take precedence when ANPs are created at the same priority, it is preferable to create ANPs at different priorities so that precedence is well defined.
```

Comment:
```suggestion
<2> The `spec.priority` field supports a maximum of 100 ANPs in the range of values `0-99` in a cluster. The lower the value, the higher the precedence because the range is read in order from lowest to highest value. Because there is no guarantee which policy takes precedence when ANPs are created at the same priority, set ANPs at different priorities so that precedence is deliberate. 
```
Thoughts/edits for clarity IMHO; also, I assume the values should be backticked? Or "precedence is as you intend" something like that